Python/linux-package-job-submission.md

```diff
@@ -0,0 +1,121 @@
+# AFL Linux fuzzing job submitted as package to MSRD
+
+This article walks you through the submission of a
+package-based fuzzing job on Linux.
+
+## Preliminary: install python and sample repository
+
+Install git and python dependencies.
+
+```bash
+# E.g. using apt-get
+sudo apt-get update
+sudo apt-get install git python3 python3-venv
+```
+
+```bash
+# Clone the MSRD sample repository
+git clone https://github.com/microsoft/msrd-rest-samples.git
+
+# Configure MSRD target instance and account
+export MSRD_TOKEN='******'
+export MSRD_URL='https://microsoftsecurityriskdetection.com'
+export MSRD_ACCOUNT='YOUR GUID'
+
+## start python
+cd ~/msrd-rest-samples/Python
+python3 -m venv venv
+. venv/bin/activate
+pip install -r requirements.txt
+```
+
+## Submit `demofuzz` on MSRD Linux Package-based job
+
+```bash
+# Submit demofuzz to MSRD
+python ./msrd.py submit -j ../SampleFuzzingJobs/DemofuzzLinux//demofuzz-ubuntu.json ../SampleFuzzingJobs/DemofuzzLinux/install-demofuzz.sh ../SampleFuzzingJobs/DemofuzzLinux/demofuzz.exe  ../SampleFuzzingJobs/DemofuzzLinux/seeds/data.bin
+```
+
+## Submit `readelf` on MSRD Linux Package-based job with AFL
+
```

Comment:
If these instructions will only work on Ubuntu, a quick note here at the top would be helpful warning the user that this requires using the experimental Ubuntu support in MSRD.

Reply:
clarified. (They are for both distributions.)